src/engine/controls/bpmcontrol.cpp

```diff
@@ -1021,6 +1030,17 @@ void BpmControl::slotBeatsTranslateMatchAlignment(double v) {
     }
 }
 
+void BpmControl::slotToggleBpmLock(double v) {
+    if (v <= 0) {
+        return;
+    }
+    const TrackPointer pTrack = getEngineBuffer()->getLoadedTrack();
+    if (!pTrack) {
+        return;
+    }
+    pTrack->setBpmLocked(not pTrack->isBpmLocked());
```

Comment:
We don't use `not`, simply negate with `!`.
```suggestion
    pTrack->setBpmLocked(!pTrack->isBpmLocked());
```
This would make it even easier to read
```suggestion
    bool locked = pTrack->isBpmLocked();
    pTrack->setBpmLocked(!locked);
```